src/main/scala/cromwell/engine/backend/jes/Run.scala

```diff
@@ -102,7 +102,7 @@ object Run  {
   private case class EventStartTime(name: String, timestamp: DateTime)
 
   def getEventList(op: Operation): Seq[ExecutionEventEntry] = {
-    val starterEvents = eventIfExists("createTime", op, "chilling out, maxing") ++ eventIfExists("startTime", op, "relaxing all cool")
+    val starterEvents = eventIfExists("createTime", op, "waiting for quota") ++ eventIfExists("startTime", op, "reticulating splines")
```

Comment:
I was thinking more semantically, like now we know "chilling out, maxing" means "waiting for quota".  Is there an analogous meaning for "relaxing all cool" like maybe "waiting for node to boot / Docker image pull / file localization"?